### PR TITLE
Add HEP for PCI-PT [CI SKIP]

### PR DESCRIPTION
**Pull request by tlehman**
**Problem:**
To enable VMs in Harvester to directly use PCI devices in the host cluster.

**Solution:**
Released in 1.1.0, PCI Passthrough is an experimental addon that enables the stories described in this HEP. This PR is to formalize the feature as an HEP, as the original HEP remained in Draft form while we shipped the feature.

**NOTE:**
- https://github.com/tlehman/harvester/blob/hep-pcipt/enhancements/20220722-pci-passthrough.md

**Related Issue:**
- https://github.com/harvester/harvester/issues/992
- https://github.com/harvester/harvester/issues/1029
- https://github.com/harvester/harvester/issues/2429

**Test plan:**
CI Disabled for this PR, since it's just static markdown documents.

### CODE REVIEW

File: enhancements/20220722-pci-passthrough.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  ### For Intel users
+```
+GRUB_CMDLINE_LINUX="intel_iommu=on iommu=pt rd.driver.pre=vfio-pci"
+```
+  ### For AMD users
+```
+GRUB_CMDLINE_LINUX="iommu=pt amd_iommu=on rd.driver.pre=vfio-pci"
+```
+
+In [installer PR 380](https://github.com/harvester/harvester-installer/pull/380) I introduced both of these parameters to avoid situations where the harv-install script had a failure before the `update_grub_settings()` function and it didn't set the correct parameters. Adding both intel and amd params is a good workaround.
```
Comment (ibrokethecloud) [inline enhancements/20220722-pci-passthrough.md]: we should update this to refer to addition of arguments via bootargs.cfg

**Comment by ibrokethecloud** [comment]
we should also cover the behavior of pcidevices controller when device is part of a iommu group.

**Comment by tlehman** [comment]
> we should also cover the behavior of pcidevices controller when device is part of a iommu group.

Done

File: enhancements/20220722-pci-passthrough.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  ### For Intel users
+```
+GRUB_CMDLINE_LINUX="intel_iommu=on iommu=pt rd.driver.pre=vfio-pci"
+```
+  ### For AMD users
+```
+GRUB_CMDLINE_LINUX="iommu=pt amd_iommu=on rd.driver.pre=vfio-pci"
+```
+
+In [installer PR 383](https://github.com/harvester/harvester-installer/pull/383) both of these parameters are introduced to /cos/bootargs.cfg avoid situations where the harv-install script had a failure before the `update_grub_settings()` function and it didn't set the correct parameters.
```
Comment (bk201) [inline enhancements/20220722-pci-passthrough.md]: nit: `/etc/cos/bootargs.cfg`, and PR 383 also solve the upgrade path.

**Review by bk201 — APPROVED** [review]
LGTM

**Review by ibrokethecloud — APPROVED** [review]
LGTM

**Pull request merged**